### ISSUE TITLE
Possible permissions error on first start

### ISSUE THREAD

**Issue by Niksko** [munnerz/kube-plex#22]
Hi, really appreciate this project, it's a great piece of work

I'm trying to get set up, but I'm currently running into some (possibly permissions related) issues when the containers start.

```
[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] 40-plex-first-run: executing... 
Creating pref shell
mkdir: cannot create directory ‘/config/Library’: Invalid argument
/var/run/s6/etc/cont-init.d/40-plex-first-run: line 82: /config/Library/Application Support/Plex Media Server/Preferences.xml: No such file or directory
chown: cannot access '/config/Library/Application Support/Plex Media Server': No such file or directory
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
Attempting to obtain server token from claim token
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     1    0     1    0     0      0      0 --:--:--  0:00:01 --:--:--     0
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
Plex Media Server first run setup complete
[cont-init.d] 40-plex-first-run: exited 0.
[cont-init.d] 50-plex-update: executing... 
failed to load external entity "/config/Library/Application Support/Plex Media Server/Preferences.xml"
[cont-init.d] 50-plex-update: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
Starting Plex Media Server.
/bin/mkdir: cannot create directory ‘/config/Library’: Invalid argument
chown: cannot access '/config/Library/Application Support': No such file or directory
```

Things I've tried:

* Creating the required folders manually. I tried creating the Library, Application Support, Plex Media Server and Codecs directories (as I made each one, it then complained about the next one), but eventually that ended up in a similar place. I would get 'Starting Plex Media Server' and then an error about being unable to create a uuid file.
* Fiddling with owners and permissions on the pvc folder in the nfs share. I tried setting the owner and group of the pvc to the docker user, and passing the PLEX_PID and PLEX_GID env variables with the docker user's values, but this didn't help. Also tried setting the permissions to umask 0777, also didn't help.

**Comment by Niksko** [comment]
Ok, after much fiddling I solved my own problem. It appears that Plex doesn't like having the config directory on an NFS share. I was using an nfs-provisioner to provision some local storage, and that's where I was getting these errors. I ripped them out and switched to using Rook (which, just fyi, doesn't seem to like btrfs) and things are working now.

This can probably be closed, and thanks again for the great project.
